Sources/CodeEditTextView/STTextViewController.swift

```diff
@@ -309,6 +312,67 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
         }
     }
 
+    private func updateCursorPosition(with textLayoutManager: NSTextLayoutManager) {
+            /// Current cursor location as NSTextLocation
+            guard let insertionPointLocation = textLayoutManager.insertionPointLocation else { return }
+
+            var lineWrapsCount = 0
+
+            // TODO: Find a more performant approach, that does not scale with line count
+            /// Count the line wraps prior to the cursor line
+            textLayoutManager.enumerateTextLayoutFragments(from: textLayoutManager.documentRange.location,
+                                                           options: [.ensuresLayout, .ensuresExtraLineFragment]
+            ) { textLayoutFragment in
```

Comment:
Please try to use multi-line indents for arguments like this:

```suggestion
            textLayoutManager.enumerateTextLayoutFragments(
                from: textLayoutManager.documentRange.location,
                options: [.ensuresLayout, .ensuresExtraLineFragment]
            ) { textLayoutFragment in
```

Reference: https://github.com/CodeEditApp/CodeEdit/wiki/Code-Style#parameter-alignment